cmake/FindZeek.cmake

```diff
@@ -89,7 +95,23 @@ macro(configure)
     endif ()
 endmacro ()
 
-function(zeek_require_version version version_number)
+macro(configure_static_build_inside_zeek)
+    set(HAVE_ZEEK yes)
+    set(ZEEK_INCLUDE_DIRS "${CMAKE_INSTALL_PREFIX}/include")
+    set(ZEEK_CMAKE_DIR "")
+    set(ZEEK_PREFIX "${CMAKE_INSTALL_PREFIX}")
+    set(ZEEK_PLUGIN_DIR "${BRO_PLUGIN_INSTALL_PATH}")
+    set(ZEEK_VERSION "${VERSION}")
+    # set(ZEEK_VERSION_NUMBER ...) # inherit from Zeek
```

Comment:
Can this go out?

Reply:
It's meant as a reminder that we do not need to set this here.